book/005-asynchronous-tooling.md

```diff
@@ -274,23 +274,23 @@ fetch('/some-endpoint/users')
   );
 ```
 
-There's nothing wrong with either approach, just make sure you and your team agrees on which method is preferred.
+There's nothing wrong with either approach, just make sure you and your team agree on which method is preferred.
```

Comment:
This is one of those weird pluralization things. Team implies multiple people, and agrees... Hmmm...

Let's meet in the middle!

```suggestion
There's nothing wrong with either approach, just make sure you and your team collectively agree on which method is preferred.
```